tower-fallback/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "tower-fallback"
 version = "0.2.40"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
-license = "MIT"
+description = "A Tower service combinator that sends requests to a first service, then retries processing on a second fallback service if the first service errors."
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
 edition = "2021"
```

Comment:
Do the dev-dependencies need versions?

Reply:
Looks like the dev-dependencies do not need versions